docs/try-flink/table_api.md

```diff
@@ -31,482 +31,290 @@ The Table API in Flink is commonly used to ease the definition of data analytics
 
-# Flink begins computing these rows as soon as 
-# as the first record for the window arrives
-> 1, 2019-01-01 01:00:00.0, $686.87
-> 2, 2019-01-01 01:00:00.0, $810.06
+From within the `table-walkthrough` folder start the docker-compose script.
 
+{% highlight bash %}
+$ docker-compose build
+$ docker-compose up -d
 {% endhighlight %}
 
-## Final Application
+You can see information on the running job via the [Flink console](http://localhost:8082/).
 
-<div class="codetabs" markdown="1">
-<div data-lang="java" markdown="1">
-{% highlight java %}
-package spendreport;
-
-import org.apache.flink.walkthrough.common.table.SpendReportTableSink;
-import org.apache.flink.walkthrough.common.table.TransactionTableSource;
-import org.apache.flink.streaming.api.TimeCharacteristic;
-import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
-import org.apache.flink.table.api.Tumble;
-import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
-
-public class SpendReport {
+![Flink Console]({% link /fig/spend-report-console.png %}){:height="400px" width="800px"}
 
-    public static void main(String[] args) throws Exception {
-        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-        env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime);
+Explore the results from inside MySQL.
 
-        StreamTableEnvironment tEnv = StreamTableEnvironment.create(env);
-
-        tEnv.registerTableSource("transactions", new UnboundedTransactionTableSource());
-        tEnv.registerTableSink("spend_report", new SpendReportTableSink());
+{% highlight bash %}
+$ docker-compose exec mysql mysql -Dsql-demo -usql-demo -pdemo-sql
 
-        tEnv
-            .scan("transactions")
-            .window(Tumble.over("1.hour").on("timestamp").as("w"))
-            .groupBy("accountId, w")
-            .select("accountId, w.start as timestamp, amount.sum")
-            .insertInto("spend_report");
+mysql> use sql-demo;
+Database changed
 
-        env.execute("Spend Report");
-    }
-}
+mysql> select count(*) from spend_report;
++----------+
+| count(*) |
++----------+
+|      110 |
++----------+
 {% endhighlight %}
-</div>
-
-<div data-lang="scala" markdown="1">
-{% highlight scala %}
-package spendreport
-
-import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
-import org.apache.flink.streaming.api.TimeCharacteristic
-import org.apache.flink.table.api._
-import org.apache.flink.table.api.bridge.scala.StreamTableEnvironment;
-import org.apache.flink.walkthrough.common.table._
-
-object SpendReport {
 
-    def main(args: Array[String]): Unit = {
-        val env = StreamExecutionEnvironment.getExecutionEnvironment
-        env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime)
+Finally, go to [Grafana](http://localhost:3000/d/FOe0PbmGk/walkthrough?viewPanel=2&orgId=1&refresh=5s) to see the fully visualized result!
```

Comment:
I haven't tried it but is `FOe0PbmGk` the same for all users?

Reply:
Yes, I ran docker compose several times it is stable